loader

```diff
@@ -161,7 +164,10 @@ def load(invocid, size, output_method, report_method, dist='gaussian', stddev=DE
         asize -= stats_msg_len
         asize = 1 if asize <= 0 else asize
         msg = stats_msg + payloadgen(asize)
-        output_method("%s%0*d%s%s" % (prefix, seq_width, seq, sep, msg))
+        if log_format == 'string':
+            output_method("%s%0*d%s%s" % (prefix, seq_width, seq, sep, msg))
+        else:
+            output_method("%s%0*d%s%s\"}" % (prefix, seq_width, seq, sep, msg))
```

Comment:
`output_method('%s%0*d%s%s"}' % (prefix, seq_width, seq, sep, msg))`

Reply:
Instead of putting logic in here, why not just override output method:
```
        if log_format == 'json':
            def output_closure(payload):
                ofp.write(payload + '\n')
        else:
            def output_closure(payload):
                ofp.write('{"message":"%s"}\n' % payload)
        output_method = output_closure
```